scaffold/tugboat/steps/2-update.sh.twig

```diff
@@ -9,6 +9,9 @@ echo "Updating..."
 composer install
 ./vendor/bin/task {{ sync_command }}
 
+# Ensure the sites directories are writable.
+chmod 755 ${DOCROOT}/sites/*/
```

Comment:
This is necessary to help fix existing sites that currently have their directories unwritable.